ENIGMAsystem/SHELL/Widget_Systems/xlib/dialogs.cpp

```diff
@@ -0,0 +1,160 @@
+/** Copyright (C) 2019 Samuel Venable
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include "dialogs.h"
+#include "Widget_Systems/widgets_mandatory.h"
+#include "Widget_Systems/General/WSdialogs.h"
+#include <string>
+using std::string;
+
+namespace enigma {
+
+bool widget_system_initialize() {
+  return true;
+}
+
+} // namespace enigma
+
+static string widget = enigma_user::ws_x11_zenity;
```

Comment:
You'll change this declaration to be `CommandLineWidgetEngine *current_widget_engine = zenity_widgets;`